react-rendering-lists/my-app/src/PokemonList.jsx

```diff
@@ -0,0 +1,11 @@
+export default function PokemonList({ pokedex }) {
+  const pokemonList = pokedex.map((pokemon) => (
+    <li key={pokemon.number}>{pokemon.name}</li>
+  ));
+
+  return (
+    <>
+      <ul>{pokemonList}</ul>
+    </>
```

Comment:
No fragment! We only need to include fragments if we're running multiple sibling elements with no parent. In this case, since we're only returning 1 `ul`, we can just return the `ul` directly.

Reply:
Keep forgetting to remove this thanks